### PR TITLE
selectable textLayer

### PR DESCRIPTION
**Pull request by mmaclach**

### CODE REVIEW

**Comment by mmaclach** [comment]
let me know how this works, I'm not certain that it will work 100%

VadimDez/ng2-pdf-viewer/#29'

File: src/pdf-viewer/pdf-viewer.component.ts [hunk 1]
```diff
@@ -170,8 +174,43 @@ export class PdfViewerComponent extends OnInit {
...
+      text.setAttribute('style', `
+                position: absolute;
+                fill: transparent;
+                line-height: 1;
+                white-space: pre;
+                cursor: text;
+            `);
+      text.textContent = textItem.str;
+      svg.appendChild(text);
+    });
```
Comment (VadimDez) [inline src/pdf-viewer/pdf-viewer.component.ts]: Looks like `svg` is never used

Comment (mmaclach) [inline src/pdf-viewer/pdf-viewer.component.ts]: updated

**Comment by VadimDez** [comment]
there're some problems with positioning, will try to fix them by myself

**Comment by mmaclach** [comment]
ok, sounds good

**Pull request merged**